service/manager.go

```diff
@@ -83,7 +83,7 @@ func newManager(builder *Builder) (Manager, error) {
 		// If the user didn't pass in a configuration provider, load the standard.
 		// Bypassing standard config load is pretty much only used for tests, although it could be
 		// useful in certain circumstances.
-		m.configProvider = config.Load()
+		m.configProvider = config.DefaultLoader.Load()
```

Comment:
While it is tempting to keep it consistent, I'd prefer to not to do that:
1. There are other methods moved that have a receiver, so it will not fully help us.
2. It will be harder to maintain in the future: config can be loaded from multiple places, instead of one, so I'd prefer to bite a bullet now.